charts/neuvector/latest/app-readme.md

```diff
@@ -0,0 +1,12 @@
+### Run-Time Protection Without Compromise
+
+NeuVector delivers a complete run-time security solution with container process/file system protection and vulnerability scanning combined with the only true Layer 7 container firewall. Protect sensitive data with a complete container security platform.
+
+NeuVector integrates tightly with Rancher and Kubernetes to extend the built-in security features for applications that require defense in depth. Security features include:
+
++ Build phase vulnerability scanning with Jenkins plug-in and registry scanning
++ Admission control to prevent vulnerable or unauthorized image deployments using Kubernetes admission control webhooks
++ Complete run-time scanning with network, process, and file system monitoring and protection
++ The industry's only layer 7 container firewall for multi-protocol threat detection and automated segmentation
++ Advanced network controls including DLP detection, service mesh integration, connection blocking and packet captures
++ Run-time vulnerability scanning and CIS benchmarks
```

Comment:
This app seems to require a `Trail License` from [dockerHub](https://hub.docker.com/_/neuvector) before launching it, otherwise, it will be failed to pull the docker images.
This should be mentioned somewhere in the app-readme page, also the `imagePullSecrets` seems to be a required parameter.

Reply:
BYOL is our current licensing model. The user will contact us either before or after they deploy the cluster. After they load the license through our UI console, the system will be fully functional.

Similarly, pull secret is not always required. Some users prefer to pull our images and push to their own registries, so it's not necessarily they have to access the dockerhub images.